modules/extensions-xml-support/src/test/java/org/mule/test/functional/LazyInitExpressionsOnConfigRefTestCase.java

```diff
@@ -34,20 +34,6 @@ public boolean enableLazyInit() {
     return true;
   }
 
-  @Ignore("W-11525405: remove ignore override once we support lazy initialization of configurations referenced by expressions")
-  @Test
-  @Override
-  public void setPayloadWithExpression() throws Exception {
-    super.setPayloadWithExpression();
-  }
-
-  @Ignore("W-11525405: remove ignore override once we support lazy initialization of configurations referenced by expressions")
-  @Test
-  @Override
-  public void setPayloadWithExpressionResolvingToIncompatibleConfigFails() throws Exception {
-    super.setPayloadWithExpressionResolvingToIncompatibleConfigFails();
-  }
-
```

Comment:
It looks like we are removing tests but we are actually unignoring inherited tests.